kit/src/main/java/com/oracle/javafx/scenebuilder/kit/skeleton/SkeletonCreator.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2016, 2021, Gluon and/or its affiliates.
```

Comment:
New files only need the current year as they didn't exist back in 2016 ;)

> Copyright (c) 2021, Gluon and/or its affiliates.

Please do the same for all new files.

Reply:
Fixed ... hopefully finally. It would be much easier without the copyright headers ;-)